modules/targets/darwin/options.nix

```diff
@@ -125,20 +131,35 @@ in {
           </warning>
         '';
       };
-      "com.apple.Safari.ContentPageGroupIdentifier.WebKit2DeveloperExtrasEnabled" =
```

Comment:
Perhaps this should be marked with `mkRemovedOptionModule` or `mkRenamedOptionModule`?

Reply:
I've added a deprecation notice to `warnings`. I initially tried with `mkRemovedOptionModule` and `mkRenamedOptionModule`, but it always triggered the warning for some reason. Might have something to do with `targets.darwin.defaults` being a freeform module.